source/chinese/chinese-compass.sh

```diff
@@ -1,7 +1,5 @@
-gmt begin chinese-compass png
+gmt begin chinese-compass pdf C-I${HOME}/.gmt/
```

Comment:
PNG is required for the HTML website.
```suggestion
gmt begin chinese-compass png,pdf C-I${HOME}/.gmt/
```